src/Controller/Organization/MissionType/MissionTypeDeleteController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller\Organization\MissionType;
+
+use App\Entity\MissionType;
+use App\Entity\Organization;
+use Doctrine\ORM\EntityManagerInterface;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+/**
+ * @Route("/delete/{id}", name="app_organization_mission_type_delete", methods={"GET"})
+ * @Security("missionType.organization == user")
```

Comment:
are you sure this security tag shouldn't be on the __invoke method ?

Reply:
I don't know what is best practice but it seems to work like that and as it is a dedicated controller (for MissionType removal) with only one __invoke method, i suppose it's ok to do like that ?

Reply:
If you're sure it's working like this, it's okay for me :-)